doc/Language/pod.rakudoc

```diff
@@ -671,76 +1170,55 @@ B<replaceable item>, a placeholder, or a metasyntactic variable. It is
 used to indicate a component of a syntax or specification that should
 eventually be replaced by an actual value. For example:
 
-=begin code :lang<rakudoc>
+=begin code
 The basic C<ln> command is: C<ln> R<source_file> R<target_file>
 =end code
 
-or:
-
-=begin code :lang<rakudoc>
-Then enter your details at the prompt:
-
-=for input
-    Name: R<your surname>
-      ID: R<your employee number>
-    Pass: R<your 36-letter password>
-=end code
+The basic C<ln> command is: C<ln> R<source_file> R<target_file>
 
 =head2 Terminal output
 
 To flag text as terminal output enclose it in C<T< >>
 =for code :lang<rakudoc>
 Hello T<John Doe>
 
-Z<If used will bust Pod::To::BigPage>
+Hello T<John Doe>
 
 =head2 Unicode
 
-To include Unicode code points or HTML5 character references in a Pod6 document,
+To include Unicode code points or HTML5 character references in a Rakudoc document,
 enclose them in C<E< >>
 
 C<E< >> can enclose a number, which is treated as the decimal Unicode
 value for the desired code point. It can also enclose explicit binary, octal,
 decimal, or hexadecimal numbers using the Raku notations for explicitly based
 numbers.
 
-It can also enclose a Unicode code point name.
+=begin code
+Raku makes considerable use of the E<171> and E<187> characters.
 
-=begin code :lang<rakudoc>
 Raku makes considerable use of the E<laquo> and E<raquo> characters.
 
-Raku makes considerable use of the E<171> and E<187> characters.
-
 Raku makes considerable use of the E<0b10101011> and E<0b10111011> characters.
 
 Raku makes considerable use of the E<0o253> and E<0o273> characters.
 
 Raku makes considerable use of the E<0d171> and E<0d187> characters.
 
 Raku makes considerable use of the E<0xAB> and E<0xBB> characters.
```

Comment:
looks like you were working against an older version and deleted these recent additions.